SPECS-EXTENDED/buildah/buildah.spec

```
@@ -40,7 +40,7 @@ BuildRequires: device-mapper-devel
 BuildRequires: golang
 BuildRequires: git
 BuildRequires: glib2-devel
-BuildRequires: glibc-static
+BuildRequires: glibc-static >= 2.35-3
```

The release component (at least in case of Mariner) always contain the `.cm[:digit:]` part as well, so it's needed, I'd suggest something like this:

```suggestion
BuildRequires: glibc-static >= 2.35-3.cm2
```

Ideally, I'd even add a macro with the required version and release at the top of the spec to make it easier to spot, in case any updates are necessary.

Having said the above, we usually ever need to set a requirement for a specific version, because release tag bumps are not expected to introduce any significant changes. If that's the case for `buildah` as well, then I'd stick to this:

```suggestion
BuildRequires: glibc-static >= 2.35
```

---

We discussed and settled on `2.35-3%{?dist}` (in case we were somehow mixing different releases of Mariner)